docs/product/components/cards.html

```diff
@@ -1,73 +1,142 @@
     <div class="stacks-preview">
 {% highlight html linenos %}
-<article class="s-card bs-sm bs-hover">…</article>
-<article class="s-card bs-md bs-hover">…</article>
-<article class="s-card bs-lg bs-hover">…</article>
+<article class="s-card bs-sm">…</article>
+<article class="s-card bs-md">…</article>
+<article class="s-card bs-lg">…</article>
 {% endhighlight %}
         <div class="stacks-preview--example">
-            <div class="grid grid__allcells4 md:fd-column gs16">
-                <article class="grid--cell s-card bs-sm bs-hover">
-                    <h1 class="fs-body3 fc-dark">Small box shadow</h1>
-                    <p class="fs-caption fc-light">This adds a small box shadow.</p>
+            <div class="grid grid__allcells4 md:fd-column gs12">
+                <article class="grid--cell s-card bs-sm">
+                    <h2 class="fs-body3 lh-sm fc-dark">Small box shadow</h2>
+                    <p class="fs-body1 fc-medium">This adds a small box shadow.</p>
                 </article>
-                <article class="grid--cell s-card bs-md bs-hover">
-                    <h1 class="fs-body3 fc-dark">Medium box shadow</h1>
-                    <p class="fs-caption fc-light">This adds a slightly larger box shadow.</p>
+                <article class="grid--cell s-card bs-md">
+                    <h2 class="fs-body3 lh-sm fc-dark">Medium box shadow</h2>
+                    <p class="fs-body1 fc-medium">This adds medium box shadow.</p>
                 </article>
-                <article class="grid--cell s-card bs-lg bs-hover">
-                    <h1 class="fs-body3 fc-dark">Large box shadow</h1>
-                    <p class="fs-caption fc-light">This is the largest box shadow.</p>
+                <article class="grid--cell s-card bs-lg">
+                    <h2 class="fs-body3 lh-sm fc-dark">Large box shadow</h2>
+                    <p class="fs-body1 fc-medium">This is the largest box shadow.</p>
                 </article>
             </div>
         </div>
     </div>
 </section>
 
+<section class="stacks-section">
+    {% header h2 | Linked Cards %}
+    <p class="stacks-copy">The <code class="stacks-code">.s-card</code> class can be applied to an <code class="stacks-code">&lt;a&gt;</code> tag for instances where a whole card should link somewhere. If possible, linked cards should visually indication that they're interactive (ex. including an <code class="stacks-code">.s-btn</code> or <code class="stacks-code">.s-link</code> somewhere).</p>
+    <p class="stacks-copy">A <code class="stacks-code">:hover</code> style for border color is automatically added to all linked cards. For linked cards with a box shadow (<code class="stacks-code">.bs-*</code>), adding a <code class="stacks-code">.bs-hover</code> class will apply a hover style to the box shadow as well.</p>
+    <div class="stacks-preview">
+{% highlight html linenos %}
+<a href="#url" class="s-card">
```

Comment:
We generally use the `…` character with dummy URLs.

Reply:
done